recipes/sos-python/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sos-python" %}
+{% set version = "0.18.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 360dd16db42664d3a609037354bd92c435a246510079621c906be9203338051c
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
```

Comment:
i think you need to depend on python directly too right?

Reply:
Do you mean adding `python` to `run`? `sos-notebook` is a python package and depends on `python` already.

Reply:
yeah, but that will make the dependency implicit. Your sos-python package depends itself on python. 

I think the test is: `If sos-notebook no longer depended on python, would sos-python still work?`

Reply:
The answer is no. `sos-notebook` depends on `python`. `sos-python` depends on `sos-notebook` so it depends on python implicitly.

Reply:
right, recepies are meant to be `explicit` in their dependencies. so python should be listed here.

Reply:
Just added.